components/clearbutton/stories/template.js

```diff
@@ -39,3 +42,42 @@ export const Template = ({
 		</div>
 	</button>
 `;
+
+// TODO: refactor this SizingGroup to use the sizing decorator instead for standardized sizing story styles
```

Comment:
Remind me because it's been a minute: are we still waiting for that sizing decorator? Is there a reason we can't use the existing one [as it's used in Treeview](https://github.com/adobe/spectrum-css/blob/main/components/treeview/stories/treeview.stories.js#L116), for instance?

Reply:
Oh excellent eyes! I didn't see that change come in. I had pulled what I could from the popover sizing function that we had liked a while back, but if we can just use this, I vote yes (it doesn't look like it has the static color options Josh created, however, these components don't need that quite yet). 

I updated the sizing stories for clear button and checkbox, so let me know if you think this works instead of waiting for that other sizing decorator: https://github.com/adobe/spectrum-css/pull/2793/commits/5e155d72aa7872e8d6e2abff39a30e56c1708255

Reply:
Yep I think it looks nice, documenting the changes here for posterity:
![image](https://github.com/user-attachments/assets/d2fe86f1-1fb5-4918-8767-259754ea2d45)
![image](https://github.com/user-attachments/assets/e39fa940-7053-46f8-9204-0851ea87c699)